### DISCUSSION TITLE
air-gap installations and nerdctl

### DISCUSSION THREAD

**Discussion by eldanh** [k3s-io/k3s#5412] [Q&A]
I was successfully using the air-gap '[Manually deploy images method](https://rancher.com/docs/k3s/latest/en/installation/airgap/#prepare-the-images-directory-and-k3s-binary)' for my application images.

However, I moved from docker-desktop to rancher-desktop (uses nerdctl) but now the 'nerdctl save' command creates a tar file in a format that k3s cannot read (the tar has the 'blobs' folder).  Previously, with docker, the tar was properly readable (the tar has the 'repositories' and 'manifest.json').

How can I instruct k3s to use the tar images in its '\var\lib\rancher\k3s\agent\images\' folder to read the 'blobs' format?
Or the other way around , to have 'nerdctl save' create a tar with the 'repositories' and 'manifest.json' format?

Thanks

**Comment by brandond** [comment]
I would suggest asking this question on the Rancher Desktop repo, or in the #rancher-desktop channel on Rancher Users Slack.
